src/rp2_common/hardware_flash/include/hardware/flash.h

```diff
@@ -46,7 +46,9 @@
 #define FLASH_SECTOR_SIZE (1u << 12)
 #define FLASH_BLOCK_SIZE (1u << 16)
 
+#ifndef FLASH_UNIQUE_ID_SIZE_BYTES
 #define FLASH_UNIQUE_ID_SIZE_BYTES 8
+#endif
```

Comment:
The description for this PR says "Support overriding FLASH_UNIQUE_ID_SIZE_BYTES in the board header."
Does that mean that in hindsight this should have become a `// PICO_CONFIG:` entry?

Reply:
Yes, probably.